### PR TITLE
Add information about accessing the Topics API model file and colab

### PR DESCRIPTION
**Pull request by samdutton**
https://deploy-preview-3244--developer-chrome-com.netlify.app/docs/privacy-sandbox/topics/

### CODE REVIEW

File: site/en/docs/privacy-sandbox/topics/index.md [hunk 1]
```diff
@@ -521,15 +521,31 @@ one or more hostnames.
...
+
+{% Img src="image/80mq7dk16vVEg8BBhsVe42n6zn82/txujKqPgnQdbwmTfdPZT.png", 
+  alt="Screenshot of chrome://topics-internal page with Classifier panel selected and tflite file path highlighted.", 
+  width="800", height="696" %}
+
+You can download the file and load the model with the [Topics Model Execution Demo colab](https://colab.sandbox.google.com/drive/1hIVoz8bRCTpllYvads51MV7YS3zi3prn). (A colab—or colaboratory—is a data 
+analysis tool that combines code, output, and descriptive text into one collaborative document.) 
+
+{% Img src="image/80mq7dk16vVEg8BBhsVe42n6zn82/OWOHUKsrvv7ZPtBP9i85.png", 
+  alt="Sceenshot of the Topics API Model Execution Demo colab.",
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/topics/index.md:547]:
```suggestion
  alt="The Topics API Model Execution Demo colab homepage.", 
```

File: site/en/docs/privacy-sandbox/topics/index.md [hunk 2]
```diff
@@ -521,15 +521,31 @@ one or more hostnames.
...
- {% endAside %}
+{% endAside %}
+ 
+#### Access the tflite classifier model file {: #access-tflite-file}
+
+The **Classifier** tab of the `chrome://topics-internals` page also provides the file path for the tflite 
+model used by the Topics API. 
+
+{% Img src="image/80mq7dk16vVEg8BBhsVe42n6zn82/txujKqPgnQdbwmTfdPZT.png", 
+  alt="Screenshot of chrome://topics-internal page with Classifier panel selected and tflite file path highlighted.",
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/topics/index.md:540]:
```suggestion
  alt="The chrome://topics-internal page with the Classifier panel open. The tflite file path is highlighted, listed under Model file path.", 
```

**Review by heyawhite — APPROVED** [review]
Small alt text suggestions

**Comment by samdutton** [comment]
> Small alt text suggestions

Ah — good call!

**Pull request merged**